lib/phoenix/channel/server.ex

```diff
@@ -233,6 +235,29 @@ defmodule Phoenix.Channel.Server do
     socket.channel.terminate(reason, socket)
   end
 
+  def fastlane(topic, %Broadcast{event: event} = msg, cache, transport_pid, channel, serializer) do
+    if channel.__fastlane__?(event) do
+      cache_key = {channel, event, serializer}
```

Comment:
we shouldn't need the event in the cache key because the cache key does not outlast an event.